src/mixins/seedable.py

```
@@ -7,49 +7,157 @@
+            >>> _ = M._seed(0, "foo")
+            >>> _ = M._seed(2, "bar")
+            >>> _ = M._seed(4, "foo")
+            >>> _ = M._seed(6, "baz")
+            >>> M._last_seed("foo")
+            (2, 4)
+            >>> M._last_seed("bar")
+            (1, 2)
+            >>> M._last_seed("baz")
+            (3, 6)
+        """
         for idx, (s, k, time) in enumerate(self._past_seeds[::-1]):
```

Ruff

Loop control variable `time` not used within loop body

Rename unused `time` to `_time`

(B007)